packages/@okta/courage-for-signin-widget/yarn.lock

```diff
@@ -127,10 +127,24 @@ array-flatten@1.1.1:
   version "1.1.1"
   resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-master/array-flatten/-/array-flatten-1.1.1.tgz#9a5f699051b1e7073328f2a008968b64ea2955d2"
 
+array-union@^1.0.1:
```

Comment:
We had a conversation about the resolved URLs before, but need to revisit this again since its been picking up traction. Can we remove this file for now?